tangelo/linq/tests/test_simulator_noisy.py

```diff
@@ -163,6 +167,10 @@ def test_noisy_simulation_qiskit(self):
         res_mixed, _ = s_nmm.simulate(circuit_mixed)
         assert_freq_dict_almost_equal(ref_mixed, res_mixed, 7.e-2)
 
+        s_nmm = get_backend(target="qiskit", n_shots=10**4, noise_model=nmm)
```

Comment:
I need a comment to describe what this tests if you're lumping it with other things

Reply:
It's just another noise model. Fixed